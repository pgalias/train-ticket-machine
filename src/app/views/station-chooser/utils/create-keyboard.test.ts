import createKeyboard from './create-keyboard';

describe('createKeyboard', () => {
  test('should return array of keyboard rows (as arrays)', () => {
    const keyboard = createKeyboard();

    expect(keyboard).toMatchSnapshot();
  });
});
