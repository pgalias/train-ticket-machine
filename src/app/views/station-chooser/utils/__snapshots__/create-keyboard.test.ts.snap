// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createKeyboard should return array of keyboard rows (as arrays) 1`] = `
Array [
  Array [
    Object {
      "modelValue": "Q",
    },
    Object {
      "modelValue": "W",
    },
    Object {
      "modelValue": "E",
    },
    Object {
      "modelValue": "R",
    },
    Object {
      "modelValue": "T",
    },
    Object {
      "modelValue": "Y",
    },
    Object {
      "modelValue": "U",
    },
    Object {
      "modelValue": "I",
    },
    Object {
      "modelValue": "O",
    },
    Object {
      "modelValue": "P",
    },
  ],
  Array [
    Object {
      "modelValue": "A",
    },
    Object {
      "modelValue": "S",
    },
    Object {
      "modelValue": "D",
    },
    Object {
      "modelValue": "F",
    },
    Object {
      "modelValue": "G",
    },
    Object {
      "modelValue": "H",
    },
    Object {
      "modelValue": "J",
    },
    Object {
      "modelValue": "K",
    },
    Object {
      "modelValue": "L",
    },
  ],
  Array [
    Object {
      "modelValue": "Z",
    },
    Object {
      "modelValue": "X",
    },
    Object {
      "modelValue": "C",
    },
    Object {
      "modelValue": "V",
    },
    Object {
      "modelValue": "B",
    },
    Object {
      "modelValue": "N",
    },
    Object {
      "modelValue": "M",
    },
  ],
  Array [
    Object {
      "modelValue": "[space]",
      "size": "lg",
      "viewValue": " ",
    },
    Object {
      "modelValue": "[backspace]",
      "size": "md",
      "viewValue": <img src="" alt="backspace icon" style="height:100%;width:100%;" />,
    },
  ],
]
`;
